src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import { Link } from 'react-router-dom'
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/Home">Home</Link>
         </div>
         <div>
-          <a href="">About</a>
+          <Link to="About">About</Link>
         </div>
         <div>
-          <a href="">Contact</a>
+          <Link to="Contact">Contact</Link>
```

Comment:
When you create Links like this, the "to" is what needs to be typed at the end of the URL  in the browser to render these components. Since your Home component is what you want to automatically come up, you would just link to="/", that way, you don't have to type anything extra at the end of the URL to get there. Then for contact and about, you would want to link to="/about" and to="/contact" so to render any of these without clicking anything, you could just type: 
yourURL.com/ to get to home,
yourURL.com/about to get to about, or
yourURL.com/contact to get to your contact page.


What you had here matches your routes, so it may have worked, the naming convention is to just be all lower-case letters and make sure you have the slashes. Otherwise, fabulous job!